### DISCUSSION TITLE
The final parsec problem of black hole binaries

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#266] [Astrophysics and Physics]
So I was reading the "list of unsolved problems in astronomy" on Wikipedia and while the list is pathetically incomplete, there were a few I was not familiar with, notably the Final parsec problem! 

I have procrastinated studying black holes, in parts because I am partly skeptical of what the concept entails and in parts because our instruments are very limited in their ability to study and characterize black holes.
Though the topic is more fruitful and promising than I thought.
One, there isn't a unique definition of what is a black hole, for example some alternative relativist or non relativist theories have slightly different concepts, such as frozen stars and they do not necessarily have a gravitational singularity.
I wonder what lesage gravity and its superluminal graviton has to say on blackholes. There is also the stochastic emitter theory which has a speed of light that depend on the source speed.
Anyway there are a few distinct speculated origins of black holes, be it supermassive black holes at the center of galaxies and maybe quasars, regular black holes, small primordial black holes and the speculated direct collapse black hole.

However black hole can be more and more studied:
What is the origin of the [M–sigma relation](https://en.m.wikipedia.org/wiki/M%E2%80%93sigma_relation) between supermassive black hole mass and galaxy velocity dispersion?[[19]](https://en.m.wikipedia.org/wiki/List_of_unsolved_problems_in_astronomy#cite_note-19)
Besides this mystery, primordial black holes should be so frequent that there should be one in the Solar system 
https://arxiv.org/abs/2409.04518

Moreover thanks to Gaia we are beginning to be able to find black holes much closer to earth 
https://en.m.wikipedia.org/wiki/Gaia_BH1
I expect GAIA DR4 to nonlinearly improve the closest black hole record. 

There is also the well popularized news that the event horizon radio interferometer has indirectly imaged (+polarization) our supermassive black hole. The next generation of the event horizon interferometer should be able to do much better imaging. 
Moreover black holes emits gravitational waves and our ability to detect them will soon considerably improve.

But the biggest unsolved problem in binary black hole merger is the final parsec problem! 
https://en.m.wikipedia.org/wiki/Binary_black_hole#Final_parsec_problem
> as the orbits shrink, the volume of space the black holes pass through reduces, until there is so little matter remaining that it could not cause a merger within the age of the universe.

> Nonetheless, supermassive black holes appear to have merged, and what appears to be a pair in this intermediate range has been observed in [PKS 1302-102](https://en.m.wikipedia.org/wiki/PKS_1302-102).[[18]](https://en.m.wikipedia.org/wiki/Binary_black_hole#cite_note-18)[[19]](https://en.m.wikipedia.org/wiki/Binary_black_hole#cite_note-19) The question of how this happens is the "final-parsec problem".[[20]](https://en.m.wikipedia.org/wiki/Binary_black_hole#cite_note-20)

Since we observe binary black hole mergers, both via gravitational waves power and via currently "too close" binary black holes, 
this is a unique independent empirical evidence that our datation of the age of the universe is wrong and that it is much older 
OR there are unclear mechanisms that attract black holes together 
While wiki list some possible theories, those that do not rely on new physics (dark matter, etc) seems to only explain a fraction of the observed rate.
Towards an empirical solution:
https://arxiv.org/abs/2308.12499
https://arxiv.org/abs/2312.11847
https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.133.121401
This SMBH abundance is also strikingly similar to estimates of nSMBH in the local Universe.
https://phys.org/news/2024-09-hubble-black-holes-early-universe.html

https://www.livescience.com/space/black-holes/james-webb-space-telescope-spots-gigantic-galaxy-starving-its-host-galaxy-to-death
https://journals.aps.org/prd/abstract/10.1103/PhysRevD.110.063533

very interesting result on black hole jet length
https://phys.org/news/2024-09-astronomers-biggest-black-hole-jets.html
 > How jets can retain such long-lived coherence is unknown at present.

**Comment by RedshiftDrift** [comment]
That's a good argument against black hole mergers!

Therefore, we **don't** *observe binary black hole mergers*!
